definitions/npm/mongoose_v4.x.x/flow_v0.50.x-/test_mongoose-v4.js

```diff
@@ -2,7 +2,7 @@
 
 import mongoose, { Schema } from "mongoose";
 
-export const AdminSchema = new Schema(
+export const AdminSchema: Mongoose$Schema<AdminDoc> = new Schema(
```

Comment:
I think that in this case you want to use type parameters like `export const AdminSchema = new Schema<Mongoose$Schema<AdminDoc>>(...` ?

Reply:
Flow added support for these few versions back.

Reply:
I just tried it like `new Schema<AdminDoc>(...)` but I get a `Missing type annotation for 'Mongoose$Schema'` again. Did not figure out why...